sleap/gui/learning/dialog.py

```diff
@@ -677,6 +683,8 @@ def view_datagen(self):
     def on_button_click(self, button):
         if button == self.save_button:
             self.save()
+        if button == self.copy_button:
+            self.copy()
```

Comment:
I think this is just an old function that doesn't even get called...
```suggestion

```

All the click events are connected to callbacks [here](https://github.com/KevinZ0217/sleap/blob/cc4ff7024e6b210457b15381cad14be22f19e29b/sleap/gui/learning/dialog.py#L147-L152):
```python
        # Connect actions for buttons
        self.copy_button.clicked.connect(self.copy)
        self.save_button.clicked.connect(self.save)
        self.export_button.clicked.connect(self.export_package)
        self.cancel_button.clicked.connect(self.reject)
        self.run_button.clicked.connect(self.run)
```